jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/test/java/org/jbpm/workbench/cm/client/list/CaseInstanceListPresenterTest.java

```diff
@@ -138,4 +151,15 @@ public void testSelectCaseInstance() {
                      dpr.getParameter(CaseOverviewPresenter.PARAMETER_CASE_ID,
                                       null));
     }
+
+    @Test
+    public void testRefreshCaskList(){
+        actions.onRefreshCaseClick(event);
+
+        verify(caseManagementService).getCaseInstances(view.getCaseInstanceSearchRequest());
+        final ArgumentCaptor<List> captor = ArgumentCaptor.forClass(List.class);
+        verify(view).setCaseInstanceList(captor.capture());
+        assertEquals(caseInstanceSummaryList.size(),
+                     captor.getValue().size());
```

Comment:
I don't see any point in re-testing the same thing which is already tested by ```testRefreshData()```.
I would keep this test as simple as possible and test only that a proper presenter method is called when the refresh button is clicked i.e.
```
@Test
    public void testRefreshCaskList(){ // Mind the typo you made in naming the method
        actions.onRefreshCaseClick(event);

        verify(presenter).refreshData(); // You would have to add @Spy annotation to presenter object
    }
```

Reply:
ok you are right